src/lib/ui/device/BatteryStatus.tsx

```diff
@@ -12,21 +12,27 @@ interface BatteryStatusProps {
 export function BatteryStatus(props: BatteryStatusProps): JSX.Element {
   return (
     <div className='seam-battery-status'>
-      <Content status={props.device.properties.battery?.status} />
+      <Content
+        status={props.device.properties.battery?.status}
+        level={props.device.properties.battery?.level}
+      />
     </div>
   )
 }
 
 function Content(props: {
   status: SeamBatteryStatus | null | undefined
+  level: number | undefined
```

Comment:
If this is allowed undefined then we need to update the string logic so it doesn't render the word undefined.

Reply:
[9b2cbd9](https://github.com/seamapi/react/pull/313/commits/9b2cbd9950f4b3248602fe422921c07d7f537cde)

Reply:
[72eafc6](https://github.com/seamapi/react/pull/313/commits/72eafc69938ed23f018c3367dcbbb7df40b2af74)